app/models/manageiq/providers/configuration_manager/inventory_group.rb

```diff
@@ -1,3 +1,9 @@
 class ManageIQ::Providers::ConfigurationManager::InventoryGroup < EmsFolder
   belongs_to :manager, :foreign_key => "ems_id", :class_name => "ManageIQ::Providers::ConfigurationManager"
+
+  virtual_column :total_configured_systems, :type => :integer
+
+  def total_configured_systems
+    Rbac.filtered(ConfiguredSystem.where(:manager_id => ems_id, :inventory_root_group_id => id)).count
```

Comment:
I think this is on the wrong class, should be InventoryRootGroup.  Then with a `has_many :configured_systems`, this could be simplified to:
```ruby
Rbac.filtered(configured_systems).count
```